python/pyproject.toml

```diff
@@ -6,6 +6,7 @@ build-backend = "maturin"
 name = "geoindex-rs"
 requires-python = ">=3.8"
 dependencies = []
+dynamic = ["version"]
```

Comment:
This says that the version key is dynamic?

Reply:
Yep - it delegates to the build tool (usually intended for git tag based releasing, but in this case, the source of truth for the version is the Cargo.toml), figured it out when setting up packaging (toml linting wouldn't shut up about the [project] section being invalid :sweat_smile: ) via https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#static-vs-dynamic-metadata